middleware/modelExport.js

```diff
@@ -298,7 +328,18 @@ module.exports = {
         asyncFn.push(helpers.getForm);
         asyncFn.push(helpers.getGrid);
 
-        async.waterfall(asyncFn, function (err, filters, form, grid) {
+        // get the actual export object
```

Comment:
When I used the example below, clicking on the 'Custom export' button opens up a modal. This is the same when the export is an array.

```
grid: {
    export: {
        label: 'Custom export',
        action: 'url/endpoint'
    }
}
```